addons/mail/static/src/models/dialog.js

```diff
@@ -11,7 +11,7 @@ registerModel({
         'attachmentImageOwnerAsAttachmentDeleteConfirm',
         'attachmentListOwnerAsAttachmentView',
         'followerOwnerAsSubtypeList',
-        'messageActionListOwnerAsDeleteConfirm',
+        'messageActionViewOwnerAsDeleteConfirm',
     ]],
     lifecycleHooks: {
```

Comment:
Don't forget to rebase master and adapt with new changes with identifying fields.

Here, it has become `identifyingMode: "xor"` and each listed field now have option `identifying: true`